### ISSUE TITLE
Container crashes immediately after start

### ISSUE THREAD

**Issue by mkosieradzki** [Microsoft/service-fabric-reverse-proxy#6]
The entire container log output is as follows
```
./startup.sh: line 26: [: _Primary_0: unary operator expected
/usr/local/bin/envoy -c config.json --service-cluster ReverseProxy --service-node _Primary_0 --mode validate
```
and it restarts immediately - I am using a clean 6.2.199.1 cluster on Azure and unmodified `reverseproxy.yml` from master branch.

I have created a PR to fix the first warning (#5), but I don't think it will be enough.

I have no further logs - but this should be trivial to reproduce.

**Comment by mkosieradzki** [comment]
I think I have forgotten to provide Secure Cluster credentials. I will give it a try.

**Comment by mkosieradzki** [comment]
OK. It seems to work now. So the only issue remaining is the warning in Container Logs fixed by #5
